scripts/builtin/imputeByKNN.dml

```diff
@@ -42,14 +42,15 @@
 #                              with M<<N and S<<N, but suboptimal imputation.
 # seed       Root seed value for random/sample calls for deterministic behavior
 #            -1 for true randomization
+# sparsity   Size of the sample for 'dist_sample' method
```

Comment:
Please, call it sample_frac to indicate the sample fraction. We want to take a sample of tuples from the input matrix, not create a sparse random matrix.